src/app/view/IAP/hooks/useClouderyOffer.ts

```diff
@@ -116,9 +117,12 @@ export const useClouderyOffer = (): ClouderyOfferState => {
       setPopupUrl(null)
     } else {
       const offersParam = formatOffers(subscriptions)
+      const lang = getLocales()[0].languageCode
```

Comment:
We have `getDeviceLanguage` that does the same job in our wrapper `src/locales/i18n.ts`

Reply:
Yep but it is not exported, so after talking with @acezard we chose to directly call `getLocales()` here instead of exporting the other method.